lib/delay.rb

```diff
@@ -6,37 +6,43 @@ class InvalidDelayExpression < ArgumentError
 class Delay
   SEPARATOR = ','.freeze
   TRANSLATIONS = {
-    'an' => :year,
-    'ans' => :year,
-    'année' => :year,
-    'années' => :year,
-    'annee' => :year,
-    'annees' => :year,
-    'year' => :year,
-    'years' => :year,
-    'mois' => :month,
-    'month' => :month,
-    'months' => :month,
-    'week' => :week,
-    'semaine' => :week,
-    'weeks' => :week,
-    'semaines' => :week,
-    'jour' => :day,
-    'day' => :day,
-    'jours' => :day,
-    'days' => :day,
-    'heure' => :hour,
-    'hour' => :hour,
-    'heures' => :hour,
-    'hours' => :hour,
-    'minute' => :minute,
-    'minutes' => :minute,
-    'seconde' => :second,
-    'second' => :second,
-    'secondes' => :second,
-    'seconds' => :second
+    fra: {
+      'an' => :year,
+      'ans' => :year,
+      'année' => :year,
+      'années' => :year,
+      'annee' => :year,
+      'annees' => :year,
+      'mois' => :month,
+      'semaine' => :week,
+      'semaines' => :week,
+      'jour' => :day,
+      'jours' => :day,
+      'heure' => :hour,
+      'heures' => :hour,
+      'minute' => :minute,
+      'minutes' => :minute,
+      'seconde' => :second,
+      'secondes' => :second
+    },
+    eng: {
+      'year' => :year,
+      'years' => :year,
+      'month' => :month,
+      'months' => :month,
+      'week' => :week,
+      'weeks' => :week,
+      'day' => :day,
+      'days' => :day,
+      'hour' => :hour,
+      'hours' => :hour,
+      'minute' => :minute,
+      'minutes' => :minute,
+      'second' => :second,
+      'seconds' => :second,
+    }
   }.freeze
-  KEYS = TRANSLATIONS.keys.join('|').freeze
+  KEYS = TRANSLATIONS[:fra].merge(TRANSLATIONS[:eng]).keys.join('|').freeze
```

Comment:
We should go for a key-agnostic approach here:
```suggestion
TRANSLATIONS.values.reduce(&:merge).keys.join('|').freeze
```
That way if we ever add another key it'll still work.

Also, since we use that "full translation hash" later on in the code, rather than re-merging each time I think we should extract it into a constant. Maybe `KEYWORDS` or `ALL_TRANSLATIONS` or something like that?